js/app.js

```diff
@@ -4,3 +4,78 @@
 console.log('App Started');
 console.log('Fruit count', fruits.length);
```

Comment:
Here is my code for how I randomized the fruit and veg arrays:

    var shuffleArray = function(array) {
    var m = array.length, t, i;
        while (m) {
            i = Math.floor(Math.random() * m--);
            t = array[m];
            array[m] = array[i];
            array[i] = t;
        }
        return array;
    }
    shuffleArray($scope.fruitAndVeg);